app/modules/munkiinfo/scripts/munkiinfo.py

```diff
@@ -106,9 +128,13 @@ def munkiinfo_report():
     if AppleCatalogURL == 'None':
         AppleCatalogURL = ''
 
+    middleware_name, middleware_version = middleware_checks()
+
     report = {
         'AppleCatalogURL': AppleCatalogURL,
         'munkiprotocol': munkiprotocol,
+        'MiddlewareName': middleware_name,
+        'MiddlewareVersion': middleware_version,
     }
```

Comment:
As we discussed on slack, it may be cleaner to do something like MiddlewareName_Version and do it in a single string other than adding two strings.